src/components/sales/OfflineOrderBill.vue

```diff
@@ -116,26 +116,16 @@ export default defineComponent({
 		}
   },
   methods:{
-		getElementTag: function(tag: keyof HTMLElementTagNameMap): string {
-			const html: string[] = [];
-			const elements = document.getElementsByTagName(tag);
-			for (let index = 0; index < elements.length; index++) {
-				html.push(elements[index].outerHTML);
-			}
-			return html.join('\r\n');
-		},
-
-		getHtmlContents: function() {
-			const printContents = document.getElementById("bill-preview");
-			return printContents && printContents.innerHTML?printContents.innerHTML:'';
+		printDiv(div_id: string, title: string, style?: string) {
```

Comment:
why you need to create a new function. 
There is already function created with this name you can call printDiv directly.